ios/NativeModules/TextileNode.m

```diff
@@ -47,6 +48,77 @@ - (dispatch_queue_t)methodQueue {
   return dispatch_queue_create("io.textile.TextileNodeQueue", DISPATCH_QUEUE_SERIAL);
 }
 
+// Export method for local photo selection
+
+RCT_EXPORT_METHOD(requestLocalPhotos:(int)minEpoch resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  NSTimeInterval seconds = minEpoch;
+  NSDate *epochNSDate = [[NSDate alloc] initWithTimeIntervalSince1970:seconds];
+
+  PHAuthorizationStatus status = [PHPhotoLibrary authorizationStatus];
+  if (status == PHAuthorizationStatusAuthorized)
+  {
+    // Setup date-string conversion
+    NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+    // Ensure date string is always in UTC
+    [dateFormatter setTimeZone:[NSTimeZone timeZoneWithAbbreviation:@"UTC"]];
+    [dateFormatter setDateFormat:@"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"];
+
+    //initialize empty local state
+    @autoreleasepool
+    {
+      //fetch all the albums
+      PHFetchOptions *onlyImagesOptions = [PHFetchOptions new];
+
+      // Limit to only photo media types
+      // NSPredicate *mediaPhotos = [NSPredicate predicateWithFormat:@"mediaType = %i", PHAssetMediaTypeImage];
+      // Limit to only photos modified later than or equal to our specified date
+      NSPredicate *minDate = [NSPredicate predicateWithFormat:@"modificationDate > %@", epochNSDate];
+      // Limit to camera roll?
+      // NSPredicate *cameraRoll = [NSPredicate predicateWithFormat:@"modificationDate > %@", epochNSDate];
+      onlyImagesOptions.predicate = minDate;
+      // Combine the multiple predicates (didn't seem to work combining them like this)
+      // [NSCompoundPredicate orPredicateWithSubpredicates:@[mediaPhotos, minDate]];
```

Comment:
Need to return to these after testing live. Not totally clear what kind of duplication happens over galleries

Reply:
moving a single file around galleries doesn't seem to create duplices (or change modified dates) so that's solid